src/store_digest.cc

```diff
@@ -381,7 +384,7 @@ storeDigestRebuildFinish(void)
 
 /* recalculate a few hash buckets per invocation; schedules next step */
 static void
-storeDigestRebuildStep(void *datanotused)
+storeDigestRebuildStep(void * /* evetData */)
```

Comment:
Typos like this one -- is one more reason not to name unused parameters in function definitions.
```suggestion
storeDigestRebuildStep(void *)
```

Reply:
removing. Thanks for spotting